src/docs/sdk/overview.mdx

```diff
@@ -229,6 +230,24 @@ The following additional headers are permitted as per CORS policy:
 - `content-encoding`
 - `transfer-encoding`
 
+### User Agent
+
+All SDKs are expected to report their name and version via the `user-agent` header. 
+The following format should be used (unless required or recommended differently by the platform, 
+e.g. for browser SDKs):
+
+`{sdk-name}/{sdk-version}`
+
+Additional information about the runtime, operating system, and others can be 
+appended as comments in parentheses, like so:
+
+`{sdk-name}/{sdk-version} ({runtime-name} {runtime-version}; {os-name} {os-version})`
```

Comment:
Should we define "; " as the expected separator between optional info? Will make it easier to parse UA logs programmatically.